src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -209,6 +248,192 @@ private UserTypeTypeParametersFormatDefinition()
         }
         #endregion
 
+        #region User Symbols - Field Name 
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.FieldName)]
+        [Name(ClassificationTypeNames.FieldName)]
+        [Order(After = PredefinedClassificationTypeNames.Identifier)]
+        [Order(After = PredefinedClassificationTypeNames.Keyword)]
+        [Order(Before = ClassificationTypeNames.StaticSymbol)]
+        [UserVisible(true)]
+        [ExcludeFromCodeCoverage]
+        private class UserSymbolsFieldNameFormatDefinition : ClassificationFormatDefinition
+        {
+            private UserSymbolsFieldNameFormatDefinition()
+            {
+                this.DisplayName = EditorFeaturesResources.User_Symbols_Field_Name;
```

Comment:
❓ Why not 'User Members' to match the existing 'User Types' items?

Reply:
I am open to a better name. =) Would 'User Members' for everything I've added (such as, Locals and Namespaces)?